scripts/server_setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# simple script to setup a server. Tested with debian 11.
+# Usage: curl <permalink> | sh
+
+sudo apt-get update && sudo apt-get install -y podman nano unzip git curl wget jq tree htop
+git clone https://github.com/piuswalter/chess-ai
+
+mkdir -p ~/.config/containers
+echo 'unqualified-search-registries = ["docker.io"]' > ~/.config/containers/registries.conf
+cat > ~/.config/containers/containers.conf << EOF
+[engine]
+events_logger = "file"
+cgroup_manager = "cgroupfs"
+EOF
+
+sudo mkdir -p /var/prtg/scripts/
+curl https://raw.githubusercontent.com/piuswalter/Chess-AI/ca4029d5095893d1bb8cecfdba08dfe2f44e86b8/scripts/games_count.sh | sudo tee /var/prtg/scripts/games_count.sh
+sudo chmod +x /var/prtg/scripts/games_count.sh
+
+# Enable 4 GiB swap space
+if ! [ -f /mnt/swapfile ]; then
+  sudo fallocate --length 4GiB /mnt/swapfile
+  sudo chmod 600 /mnt/swapfile
+  sudo mkswap /mnt/swapfile
+  sudo swapon /mnt/swapfile
+  sudo sh -c 'echo "/mnt/swapfile swap swap defaults 0 0" >> /etc/fstab'
+  sudo sh -c 'echo "vm.swappiness=10" >> /etc/sysctl.conf'
+fi
```

Comment:
Use `echo <content> | sudo tee -a <file>` to append data to a file without write privileges.
A fail-safe implementation which does not bloat the given files after repeated execution would look like this:

```suggestion
function replace_in_file() {
  if [ -z "$3" ]; then
    echo "Usage: line_in_file <file> <regex_start> <value>"
    exit 1
  fi
  if [ -z "$(grep ^$2 $1)" ]; then
    # append line to given file
    echo "$3" | sudo tee -a "$1"
  else
    # line already exists in file, replace it
    sudo sed -i "s|$2.*|$3|g" "$1"
  fi
}

# Enable 4 GiB swap space
if ! [ -f /mnt/swapfile ]; then
  sudo fallocate --length 4GiB /mnt/swapfile
  sudo chmod 600 /mnt/swapfile
  sudo mkswap /mnt/swapfile
  sudo swapon /mnt/swapfile
  replace_in_file /etc/fstab "/mnt/swapfile"  "/mnt/swapfile swap swap defaults 0 0"
  replace_in_file /etc/sysctl.conf "vm\.swappiness=" "vm.swappiness=10"
fi
```